include/swift/AST/DiagnosticsSema.def

```diff
@@ -5606,6 +5606,11 @@ ERROR(property_wrapper_dynamic_self_type, none,
       "property wrapper %select{wrapped value|projected value}0 cannot have "
       "dynamic Self type", (bool))
 
+ERROR(property_wrapper_var_must_be_static, none,
+      "property %0, must be static because property wrapper %1 can only "
+      "be applied to static properties",
```

Comment:
Might be simpler to just say: "property wrapper ___ can only be applied to static properties"

Reply:
Hmm, I wanted to include the name of the property, you sure we'd prefer the shorter message?

Reply:
I think the name of the property isn't needed, because the error message will already be pointing to the var decl with the problematic wrapper on it, whether in the terminal or xcode.